src/main/java/uk/gov/companieshouse/exemptions/controller/ControllerExceptionHandler.java

```diff
@@ -49,15 +51,15 @@ public ResponseEntity<Void> handleServiceUnavailable(ServiceUnavailableException
     @ExceptionHandler(value = {BadRequestException.class, DateTimeParseException.class,
             HttpMessageNotReadableException.class, MethodArgumentNotValidException.class})
     public ResponseEntity<Void> handleRequestAndParseError(Exception ex) {
-        LOGGER.error("Invalid request body", ex);
+        LOGGER.error(ex.getMessage(), DataMapHolder.getLogMap());
         return ResponseEntity
                 .status(HttpStatus.BAD_REQUEST)
                 .build();
     }
 
     @ExceptionHandler(value = {Exception.class, SerDesException.class})
     public ResponseEntity<Void> handleInternalServerError(Exception ex) {
-        LOGGER.error(ex.getClass().getName(), ex);
+        LOGGER.error(ex.getMessage(), DataMapHolder.getLogMap());
```

Comment:
Error level is fine here as it may not be recoverable as this is the default exception handler case. Still needs the exception in the log params though.